libp2p/types/libp2p.go

```diff
@@ -0,0 +1,33 @@
+//go:build !production
+
+package types
+
+//go:generate mockgen -source=$GOFILE -destination=./mocks/libp2p_mock.go
+
+import (
+	"context"
+
+	"github.com/libp2p/go-libp2p/core/host"
+)
+
+type Host interface {
+	host.Host
+}
+
+//type Message interface {
+//	// TODO: what?
+//}
+
+type Subscription interface {
+	Topic() string
+	Next(ctx context.Context) (interface{}, error)
```

Comment:
s/interface{}/any

Ditto elsewhere

Reply:
Nice, I wasn't aware that was thing - thanks! This wasn't supposed to be committed though, I don't think we actually need it.